.github/build-x86_64-apple-darwin

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+echo ✨ System Info  
+sw_vers
+echo ✨ System Details
+uname -a
+echo ✨ LLVM Version
+llvm-config --version
+
+echo ✨ CMAKE_CXX_COMPILER=$CMAKE_CXX_COMPILER
+echo ✨ CMAKE_C_COMPILER=$CMAKE_C_COMPILER
+
+# llvm@11 paths
+export CC=/usr/local/opt/llvm@11/bin/clang
+export CXX=/usr/local/opt/llvm@11/bin/clang++
+
+export PATH="/usr/local/opt/llvm@11/bin:$PATH"
+export LDFLAGS="-L/usr/local/opt/llvm@11/lib"
+export CPPFLAGS="-I/usr/local/opt/llvm@11/include"
+
+echo ✨ clang -v
+clang -v
+echo ✨ clang++ -v
+clang++ -v
+
+echo ✨ LD_LIBRARY_PATH=$LD_LIBRARY_PATH
+
+echo ✨ CC=$CC
+echo ✨ CXX=$CXX
+echo ✨ LIBS=$LIBS
+
+# fix clang path's to bb expectations
+sudo mkdir -p /usr/local/opt/llvm/bin/
+sudo ln -s $(which $CC) /usr/local/opt/llvm/bin/clang

```

Comment:
I see you a linking here to match what bb expects. Just taking note of that once we update the barretenberg reference in `aztec_backend` we may be able to remove this linking in the future and just specify the `CC` and `CXX` as expected in this PR: https://github.com/AztecProtocol/barretenberg/pull/100

Reply:
agreed - that CMake configuration should get improved and then we would fix it here as well